CONTRIBUTING.md

```diff
@@ -121,7 +121,7 @@ requirements for inclusion in the package:
 
 **Code Quality**
   * Are the [coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html) followed?
-  * Is the code compatible with Python >=3.8?
+  * Is the code compatible with Python >=3.10?
```

Comment:
If you want the Python version written here to be one that is actually supported by `astropy` then I recommend you write the latest supported version (i.e. 3.12) here. That way this page stays valid for longer.

Reply:
Or the more generic "compatible with our supported versions of Python (see pyproject.toml)"

Reply:
✅